flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -95,6 +95,9 @@ def _get_package_source(self, default_version=None):
         unsupported={backends.LOOPBACK},
         reason="Does not maintain compute_instance_id across restarting "
                "flocker (and didn't as of most recent release).")
+    @skip_backend(
+        unsupported={backends.GCE},
+        reason="GCE was not available during the most recent release.")
```

Comment:
:question: Worth an XXX and a FLOC issue reminder? I guess you'll remember to re-enable this after the next marketing release.